ruff.toml

```diff
@@ -26,10 +28,6 @@ ignore = [
 ]
 extend-ignore = [
 	"UP015",  # redundant-open-modes, explicit is preferred
```

Comment:
This can be moved in `ignore`. `extend-ignore` is deprecated https://docs.astral.sh/ruff/settings/#lint_extend-ignore